cmake/Scripts/configure

```diff
@@ -48,6 +48,7 @@ Optional Packages:
   --with-superlu=PATH     user defined path to SuperLU library
   --with-umfpack=PATH     use UMFPACK/SuiteSparse from a specified location
   --with-ert=PATH         Use ERT libraries
+  --with-zoltan=PATH      Use ZOLTAN libraries
```

Comment:
I think I'd like a little explanation of `PATH` in the summary here--e.g.,
```
Use ZOLTAN libraries installed in directory named by PATH
```
or something of the sort.